index.bs

```diff
@@ -551,34 +642,42 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
-    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
+    {{CredentialCreationOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
 
     <ol class="algorithm">
-      1.  Let |settings| be the <a>current settings object</a>
+        1.  Let |settings| be the <a>current settings object</a>
+
+        1.  Let |interface objects| be the set of [=interface objects=] on |settings|'
+            [=environment settings object/global object=].
+
+        1.  Let |relevant interface objects| be an [=set/empty=] [=set=].
+
+        1.  For each |object| in |interface objects|:
 
-      2.  Let |interface objects| be the set of [=interface objects=] on |settings|'
-          [=environment settings object/global object=].
-         
-      3.  Let |relevant interface objects| be an empty set.
-      
-      4.  For each |object| in |interface objects|:
+            1.  If |object|'s [=inherited interfaces=] do not <a for="set">contain</a> {{Credential}},
+                <a for="iteration">continue</a>.
 
-          1.  If |object|'s [=inherited interfaces=] do not <a for="set">contain</a> {{Credential}},
-              <a for="iteration">continue</a>.
+            1.  Let |credentialType| be |object|'s {{Credential/[[type]]}} slot's value.
 
-          2.  Let |key| be |object|'s {{Credential/[[type]]}} slot's value.
+            1.  Let |key| be the result of looking up the |credentialType|'s |options|'s
```

Comment:
Looking up the options member in the registry would return a whole row, not just the Options Member Identifier column.

Reply:
Well, the intent is that the terminology:
> ...the result of looking up the credentialType’s options’ member identifier in the registry...

...returns the value in the registry's "options member identifier" column, for the given credentialType.  No?

In looking at [performance-timeline](https://www.w3.org/TR/performance-timeline/#performance-timeline)'s use of it's [registry](https://w3c.github.io/timing-entrytypes-registry/#registry), it makes simple algorithm statements that index into its registry, like so:
> * An integer maxBufferSize, initialized to the [registry](https://w3c.github.io/timing-entrytypes-registry/#registry) value for this entry type.
> * A boolean availableFromTimeline, initialized to the [registry](https://w3c.github.io/timing-entrytypes-registry/#registry) value for this entry type.

...so maybe we could state...
> Let |key| be the options’ member identifier registry value for the credentialType's registry entry.

..is that more clear?

Reply:
How about

```suggestion
            1.  Let |key| be the [=Credential Type Registry/Options Member Identifier=] whose [=Credential Type Registry/Credential Type=] is |credentialType|.
```

With both of those terms `<dfn>`ed in the `<th>` cell of the registry table.